dagstore.go

```diff
@@ -318,6 +333,11 @@ func (d *DAGStore) Start(ctx context.Context) error {
 	return nil
 }
 
+// TODO: There is a race between calling this and deal expory which will cause deletion from the inverted index.
```

Comment:
I think we decided we don't need to worry about this race now, right? We can probably just remove this comment